website/src/components/discourse/index.js

```diff
@@ -0,0 +1,122 @@
+import React, { useState, useEffect } from 'react'
+import axios from 'axios'
+import feedStyles from './styles.module.css';
+
+export default function DiscourseFeed({
+  order = 'latest_topic',
+  status = undefined,
+  after = undefined,
+  before = undefined,
+  inString = undefined, 
+  min_posts = undefined,
+  max_posts = undefined,
+  min_views = undefined,
+  max_views = undefined,
+  tags = undefined, 
+  term = undefined,
+  category = undefined,
```

Comment:
Can we default this to `'help'` so we don't have to specify it everywhere? That's going to be the overwhelming majority of embeds.